watertap/core/plugins/tests/test_solvers.py

```diff
@@ -69,6 +69,11 @@ def test_pyomo_registration(self, s):
     def test_idaes_registration(self):
         assert get_solver().__class__ is IpoptWaterTAP
 
+    @pytest.mark.unit
+    @pytest.mark.requires_idaes_solver
+    def test_attribute_passthrough(self, s):
+        assert s.available()
+
```

Comment:
It might make sense to also test the non-existing-attribute code path:

```suggestion
    @pytest.mark.unit
    @pytest.mark.requires_idaes_solver
    def test_attribute_passthrough(self, s):
        assert s.available()
        with pytest.raises(AttributeError, match=".*this_attribute_should_not_exist.*"):
            s.this_attribute_should_not_exist
```